forms-flow-web-root-config/src/serviceWorkerSetup.js

```diff
@@ -0,0 +1,56 @@
+export async function register(config) {
+  // Check if service workers are supported by the browser
+  if ('serviceWorker' in navigator) {
```

Comment:
need to check this condition in safari and firefox. because some option will not work with all browsers like copyToClipboard from navigator will work with chorm but not work with safari

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Navigator/serviceWorker

![image](https://github.com/AOT-Technologies/forms-flow-ai/assets/61222674/7f099e3b-31a3-4374-b287-bb82163a4c4a)